std/os/wasi.zig

```diff
@@ -0,0 +1,105 @@
+use @import("wasi/core.zig");
+
+pub const STDIN_FILENO = 0;
+pub const STDOUT_FILENO = 1;
+pub const STDERR_FILENO = 2;
+
+pub const ESUCCESS = 0;
```

Comment:
Are these (and stdio constants above) defined somewhere? Might be nice to have a reference link if you have it handy.

Reply:
The currently available sources are https://github.com/CraneStation/wasmtime/blob/master/docs/WASI-api.md and https://github.com/CraneStation/wasi-sysroot/blob/wasi/libc-bottom-half/headers/public/wasi/core.h.

The [`liftoff` branch of WebAssembly/WASI](https://github.com/WebAssembly/WASI/tree/liftoff) is probably going to be merged into master and might be the authoritative source in the future.